.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/devcontainers/base:jammy
```

Comment:
Let's target 20.04 instead of 22 since that's what platform and streets are targeting right now.

Reply:
Done